ci/do_ci.sh

```diff
@@ -8,48 +8,50 @@ set -e
 echo "building using ${NUM_CPUS} CPUs"
 
 function bazel_release_binary_build() {
-  cd "${ENVOY_CONSUMER_SRCDIR}"
   echo "Building..."
-  bazel --batch build ${BAZEL_BUILD_OPTIONS} -c opt @envoy//source/exe:envoy-static.stripped.stamped
+  cd "${ENVOY_CI_DIR}"
+  bazel --batch build ${BAZEL_BUILD_OPTIONS} -c opt //source/exe:envoy-static.stripped.stamped
   # Copy the envoy-static binary somewhere that we can access outside of the
   # container for building Docker images.
   cp -f \
-    "${ENVOY_CONSUMER_SRCDIR}"/bazel-genfiles/external/envoy/source/exe/envoy-static.stripped.stamped \
+    "${ENVOY_CI_DIR}"/bazel-genfiles/source/exe/envoy-static.stripped.stamped \
     "${ENVOY_DELIVERY_DIR}"/envoy
 }
 
 if [[ "$1" == "bazel.release" ]]; then
   echo "bazel release build with tests..."
   bazel_release_binary_build
   echo "Testing..."
-  bazel --batch test ${BAZEL_TEST_OPTIONS} -c opt --test_output=all \
-    --cache_test_results=no @envoy//test/... //:echo2_integration_test
+  bazel --batch test ${BAZEL_TEST_OPTIONS} -c opt //test/...
+  rm -f bazel-*
```

Comment:
It was a small nod to the notion that someone might run `bazel.dev` and then `bazel.coverage`. The annoying circular symlinks will then cause problems for `bazel.coverage`. Let me move this into the `bazel.coverage` target to avoid it hurting incremental builds.